### PR TITLE
Pin ginkgo version in Dockerfile.tools

### PR DESCRIPTION
**Pull request by jlojosnegros**
The build system is trying to download and compile the latest ginkgo
version, which is too new and doesn't work anymore with the golang
version we use

### CODE REVIEW

File: openshift-ci/Dockerfile.tools [hunk 1]
```diff
@@ -6,6 +6,8 @@ ENV GOVERSION=1.16.3
 ENV GOPATH=/go
 ENV GOBIN=${GOPATH}/bin
 ENV PATH=${PATH}:${GOROOT}/bin:${GOBIN}
+ENV GO111MODULE=on
+ENV GINKGO_VERSION v1.12.1
```
Comment (yanirq) [inline openshift-ci/Dockerfile.tools:10]: why is the convention here is different from the rest ?
`ENV GINKGO_VERSION v1.12.1`
and not `ENV GINKGO_VERSION=v1.12.1 ` ? 
(might not have a negative affect but still)

Comment (jlojosnegros) [reply openshift-ci/Dockerfile.tools:10]: I have just used the same line as in the previous branches, so no strong reason so far.
Will change to keep it homogeneous.

Comment (fromanirh) [reply openshift-ci/Dockerfile.tools]: Well, it is a bit annoying indeed but it's cosmetic more than functional: https://docs.docker.com/engine/reference/builder/#env (see: "alternatyve syntax")
next time let's be consistent with the surrounding code, but it's not a big deal (and PAO is already too messy already; OTOH that's not a reason to increase the mess...)

**Comment by fromanirh** [comment]
/override ci/prow/e2e-gcp
/override ci/prow/e2e-gcp-operator-upgrade

overrides must be used sparingly. In this case it's ok because we are 100% sure this specific change cannot possibly affect production code, thus causing test failures.

**Pull request merged**